common/api/core-geometry.api.md

```diff
@@ -778,9 +777,6 @@ export class BSplineCurve3d extends BSplineCurve3dBase {
     evaluatePointAndDerivativeInSpan(spanIndex: number, spanFraction: number): Ray3d;
     evaluatePointInSpan(spanIndex: number, spanFraction: number): Point3d;
     extendRange(rangeToExtend: Range3d, transform?: Transform): void;
-    fractionToPoint(fraction: number, result?: Point3d): Point3d;
```

Comment:
It looks like you have deleted some `@public` methods - or do I misread?

Reply:
Think it's okay and override wasn't needed as implementation was identical to BSplineCurve3dBase base class?

Reply:
I removed override methods that have the same implementation as super. So there should be a super class implementation that survives. In this case, BSplineCurve3dBase.fractionToPoint() is inherited by BSplineCurve3d.